src/Lecture4.hs

```diff
@@ -267,7 +369,7 @@ Use functions 'readFile' and 'putStrLn' here.
 -}
 
 printProductStats :: FilePath -> IO ()
-printProductStats = error "TODO"
+printProductStats = putStrLn . calculateStats <=< readFile
```

Comment:
Nice usage of the "fish" operator! 🐟

Reply:
I read about it and this is the first time I have the chance to use it. 
It's like pure composition but with effects. It reads pretty nice. 😄